book/docs/classes/OpusPool/Constructor/constructor.md

```diff
@@ -0,0 +1,24 @@
+### constructor
+
+• **new OpusPool**(`params`): [`OpusPool`](../../OpusPool.md)
+
+Creates an OpusPool instance
+
+#### Parameters
+
+| Name | Type | Description |
+| :------ | :------ | :------ |
+| `params` | `Object` | Initialization parameters |
+| `params.address` | \`0x$\{string}\` | An address of the currently connected user wallet. If the user connects to a different wallet, OpusPool must be re-instantiated with a new user address |
```

Comment:
Maybe the `0x${string}` can be replaced with Hex, the formatting doesn't work for `0x${string}`

Reply:
Like \`\`\`test\`\`\`

Reply:
This is generated automatically by typedoc, I fixed it by postprocessing